ci-scripts/copy_to_archive

```diff
@@ -33,42 +32,53 @@ fi
+    for f in ${LOG_DIR}/cooker/**/console-latest.log; do
+      echo "Archiving console-latest.log file to $ARCHIVE_DIR"
+      mv "$f" $ARCHIVE_DIR
+    done
+fi
+
+# Copy license manifest file
+if [ -f ${LICENSE_DIR}/${IMAGE_BASENAME}-*20*/license.manifest ]; then
+    echo "Archiving license.manifest file to $ARCHIVE_DIR"
+    mv ${LICENSE_DIR}/${IMAGE_BASENAME}-*20*/license.manifest $ARCHIVE_DIR
 fi
 
-# If there is an Intel directory
-if [ -d ${IMAGE_DIR}/${INTEL_ARCH} ]; then
-    echo "Archiving Intel image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${INTEL_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+# Copy if image files exist
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
 fi
 
-# If there is an ARP directory
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bmap ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic.bmap file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
+fi
 
-if [ -d ${IMAGE_DIR}/${ARP_ARCH} ]; then
-    echo "Archiving ARP image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${ARP_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bz2 ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.wic.bz2 file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
 fi
 
-# If there is an RPI directory
-if [ -d ${IMAGE_DIR}/${RPI_ARCH} ]; then
-    echo "Archiving RPI image to $ARCHIVE_DIR"
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*rootfs.wic $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bmap $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*rootfs.wic.bz2 $ARCHIVE_DIR
-    mv ${IMAGE_DIR}/${RPI_ARCH}/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*20*.swu ]; then
+    echo "Archiving ${IMAGE_BASENAME}.swu file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*20*.swu $ARCHIVE_DIR
+fi
+
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.manifest ]; then
+    echo "Archiving ${IMAGE_BASENAME}-rootfs.manifest file to $ARCHIVE_DIR"
+    mv ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*rootfs.manifest $ARCHIVE_DIR
 fi
 
 # If there is a Jetson TX2 directory
-if [ -d ${IMAGE_DIR}/${JETSON_ARCH} ]; then
+if [ -f ${IMAGE_DIR}/**/${IMAGE_BASENAME}-*.tegraflash.zip ]; then
```

Comment:
This artifact is specific to NVidia (Jetson TX2) and there is no reason to loop through all variants.